### PR TITLE
Add lbr state smoothing node

### PR DESCRIPTION
**Pull request by cmower**

### CODE REVIEW

**Comment by mhubii** [comment]
looks legit. Consider moving this into a package called `lbr_fri_ros2`, because this is where future logic will live

File: lbr_bringup/scripts/lbr_state_smoothing_node.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        self.declare_parameter('max_window_length', 20)
+        self._max_window_length = int(self.get_parameter('max_window_length').value)
+        self._window = []
+
+        # Set filter method
+        self._filter = self._filter_mean
+        # self._filter = self._filter_median
+
+        # Create publishers/subscribers
+        self._lbr_state_smooth_publisher = self.create_publisher(LBRState, "/lbr_state/smooth", 1)
```
Comment (mhubii) [inline lbr_bringup/scripts/lbr_state_smoothing_node.py]: NIT: subscriber first, then publisher (logical order)

Comment (cmower) [reply lbr_bringup/scripts/lbr_state_smoothing_node.py]: disagree: the subscriber can be called before the publisher is created - I've had issues with this in past (maybe its a solved issue in ros2)

**Comment by cmower** [comment]
> Consider moving this into a package called `lbr_fri_ros2`, because this is where future logic will live

Do you mean the `lbr_fri_ros2_stack` package? Otherwise, seems a bit overkill to create a whole package for one node.

File: lbr_bringup/scripts/lbr_state_smoothing_node.py (hunk 1 above)
Comment (mhubii) [reply lbr_bringup/scripts/lbr_state_smoothing_node.py]: oh okay true, sorry

**Comment by mhubii** [comment]
> 

a lot of the new logic moves into this `lbr_fri_ros2` package and this smoothing node fits nicely into it

**Comment by cmower** [comment]
Updates ready to merge?

**Pull request merged**